CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing
+
+Welcome, if you're looking to help, this document is a great place to start! 
+
+
+# Finding Things That Need Help
+
+If you're looking to help, this is a great place to start. 
+- Read through the [MDM Protocol Reference](https://developer.apple.com/library/content/documentation/Miscellaneous/Reference/MobileDeviceManagementProtocolRef/3-MDM_Protocol/MDM_Protocol.html) on the Apple website. Having a deeper understanding of MDM can help with designing features and uncovering bugs. 
+- Follow the [Quickstart](https://github.com/micromdm/micromdm/wiki/Quickstart) guide and make edits if something doesn't look right. 
+- If you run into a problem that you're not sure how to fix, file a bug. 
+- Browse through the open issues. We try to tag issues as [**beginner friendly**](https://github.com/micromdm/micromdm/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner-friendly) where appropriate.
+
+# Building the project
+
```

Comment:
Since MicroMDM won't build properly with outdated versions of Go I think a note that says use version 1.8 would be appropriate.